src/database/mongo.js

```diff
@@ -17,5 +19,5 @@ async function getDatabase() {
 
 module.exports = {
   getDatabase,
-  startDatabase,
+  startDatabase
```

Comment:
apparently, you haven't changed a thing here... so why are you reformatting this file?

Reply:
Oh, yeah. This must have been from prettier.